hw/ip/csrng/data/csrng.hjson

```diff
@@ -450,5 +509,26 @@
         }
       ]
     },
+    { name: "FORCE_ERR_CODE",
```

Comment:
this is a nit pick, but would `ERR_CODE_TEST` be a better name?

Reply:
Yes, that looks good also - seems consistent with the ALERT_TEST name. I think I will change it.